data/sql/updates/pending_db_world/fix-orphic-bracers-loot.sql

```diff
@@ -0,0 +1,6 @@
+DELETE FROM `reference_loot_template` WHERE `Item` = 18337;
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
```

Comment:
I hadn't noticed it before.
Missing DELETE before INSERT in the `creature_loot_template` table

Reply:
It has been added! :D